app/models/hero.rb

```diff
@@ -0,0 +1,25 @@
+class Hero
+  attr_reader :title, :image, :image_2x
+
+  def initialize(data)
+    @title = data["title"] || ""
+    @image = data["image"] || ""
+    @image_2x = data["image_2x"] || ""
+    @enabled = data["enabled"]
+  end
+
+  def enabled?
+    @enabled
+  end
+
+  def image_src
+    @image || @image_2x
+  end
+
+  def image_srcset
+    srcset = {}
+    srcset["1x"] = @image if @image
```

Comment:
[suggestion] this will always be true, because if `data["image"]` is either `nil` or `false` then `@image` will be set to an empty string, and an empty string will evaluate here to be truthy

```suggestion
    srcset["1x"] = @image if @image.present?
```